### PR TITLE
Added icon-block bean type

### PR DESCRIPTION
**Pull request by cynmij**

### CODE REVIEW

**Comment by sherakama** [comment]
Oh man, I completely forgot to add the bean class for the icon block. It seems that I have blown this work away already. Drat. I will recreate.

**Comment by sherakama** [comment]
@cynmij,

For this new block can you also add a set of permissions into the stanford_bean_types_permisisons feature. Thank you.

File: stanford_bean_types.info [hunk 1]
```diff
@@ -124,6 +135,9 @@ features[field_instance][] = bean-stanford_testimonial_block-field_s_image_info
 features[field_instance][] = bean-stanford_testimonial_block-field_s_testimonial_credits
 features[field_instance][] = bean-stanford_testimonial_block-field_s_testimonial_quote
 features[field_instance][] = bean-stanford_testimonial_block-field_testimonial_more_link
+features[variable][] = ds_extras_field_template
```
Comment (sherakama) [inline stanford_bean_types.info]: Please do not include this strongarm variable.

File: stanford_bean_types.strongarm.inc [hunk 2]
```diff
@@ -13,6 +13,13 @@ function stanford_bean_types_strongarm() {
   $strongarm = new stdClass();
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'ds_extras_field_template';
```
Comment (sherakama) [inline stanford_bean_types.strongarm.inc]: Please remove this strongarm variable.

**Comment by cynmij** [comment]
screen shot 2015-09-16 at 12 01 01 pm

**Comment by sherakama** [comment]
@cynmij,

In that last pull request you have removed the testimonial bean. Please restore it.

Also, If this module relies on that ds_contrib module I will need to pull the relevant code into this module as I don't want this to be dependant on an unsupported module. I am gald to hear that it worked.

**Comment by cynmij** [comment]
I'm not using testimonial bean anymore.  I created a testimonial view instead.

**Comment by sherakama** [comment]
@cynmij,

The testimonial block has been around in this module for a while now. I don't think we can just remove it. I think that would be very bad. I'll bring it up in scrum.

**Pull request merged**